doc/api/esm.md

```diff
@@ -498,13 +528,6 @@ In most cases, the ES module `import` can be used to load CommonJS modules.
 If needed, a `require` function can be constructed within an ES module using
 [`module.createRequire()`][].
 
-#### No `__filename` or `__dirname`
-
-These CommonJS variables are not available in ES modules.
-
-`__filename` and `__dirname` use cases can be replicated via
-[`import.meta.url`][].
-
```

Comment:
I don't think we can fully delete this section; we need to edit it to explain that the variables `__filename` and `__dirname` don't exist as pseudo-globals but do exist as `import.meta.filename` and `import.meta.dirname`.

Reply:
This still needs to be addressed